97.md

```diff
@@ -0,0 +1,25 @@
+NIP-97
+======
```

Comment:
Couldn't this just be added to NIP-98?

Reply:
Yes.

Reply:
I think they are best as separate NIPs, because a client might want to implement NIP-98 for authenticated communication with a server (for any particular purpose), without needing to necessarily support to Nostr Login flows.